entries/c/cam.ac.uk.json

```
@@ -0,0 +1,27 @@
+{
+  "University of Cambridge": {
+    "domain": "cam.ac.uk",
+    "img": "cam.ac.uk.png",
+    "tfa": [
+      "totp",
+      "sms",
+      "custom-software",
+      "custom-hardware",
+      "call"
+    ],
+    "custom-software": [
+      "Microsoft Authenticator",
+      "Authy"
```

While Authy is mentioned, it's only used in the context of a TOTP RFC-6238 application and no methods specific to Authy are supported.
```suggestion
      "Microsoft Authenticator"
```